client/js/render.js

```diff
@@ -24,37 +24,73 @@ module.exports = {
 
 function buildChannelMessages(data) {
 	return data.messages.reduce(function(docFragment, message) {
-		appendMessage(docFragment, data.id, data.type, message.type, buildChatMessage({
```

Comment:
Just curious, why did you move these from arguments to an object? Also, any interest in using parameter destructuring in `appendMessage` (`function f({ a, b }) { ... }`)?